src/components/Tabs/Tabs.jsx

```diff
@@ -2,0 +1,32 @@
+import classNames from 'classnames/bind';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```

Comment:
Update the variable `selectedTab` to use `activeTabId` instead of `selectedTabId` to match the updated prop name.